lib/vdsm/api/vdsm-api.yml

```diff
@@ -8451,6 +8451,21 @@ types:
             name: path
```

Comment:
Notes about the commit message: I don't think we need these ordered lists, since these
items are not ordered. A simpler way to describe this change is:

    Add domainID, imageID, and volumeID to {File,Block}ScratchDisk.

This describe what this change does, but we need to understand why, and how the ids will be used.